docs/src/parametric.md

```diff
@@ -32,4 +36,34 @@ UnequalVarianceZTest
 ## F-test
 ```@docs
 VarianceFTest
-```
\ No newline at end of file
+```
+
+## One-way ANOVA Test
+
+```@docs
+OneWayANOVATest
+```
+
+Example:
```

Comment:
We don't show examples for other tests AFAIK. Can't you put it in the docstring instead?